docs/decisions/0018-units-as-containers.rst

```diff
@@ -0,0 +1,67 @@
+18. Modeling Units as a Concrete Implementation of the Container Capability
```

Comment:
I did this exercise, in which I instanced the decisions made for containers to be for units and components, and it's way easier to understand!